src/objc.ts

```diff
@@ -370,6 +370,7 @@ export interface Class {
   baseClassName:string;
   classMethods:Method[];
   comments:Comment[];
+  generics:string[];
```

Comment:
Could we name this something like `covariantTypes` or `covariantGenericTypes`? `generics` seems too... generic? :P